.github/CODEOWNERS

```diff
@@ -77,7 +77,10 @@ Jenkinsfile.github                                       @Cray-HPE/csm-devops
 # IUF Items
 /operations/iuf/                                         @Cray-HPE/IUF
 
+#Observability Items
+/operations/observability                                @Cray-HPE/cray-monitoring
+
 # Include SSI on these CSM items.
-/install/                                                @Cray-HPE/docs-csm-reviewers @Cray-HPE/metal @Cray-HPE/ssi-reviewers
```

Comment:
The commit message does not indicate why this is changing at all for `/install`/, `/operations/`, and `/upgrade/`. It seems like a bad rebase/merge was done.

Reply:
Yes, it seems like the wrong merge. I have reverted the change and the only change required is added.